0-ad-history-mod-package/simulation/templates/structures/athen/civil_centre.xml

```diff
@@ -7,9 +7,9 @@
     <Civ>athen</Civ>
     <SpecificName>Agora</SpecificName>
     <History>
-    	Originally, Athens was ruled by a line of kings, but, like many other Greek city states, eventually became a republic. Athens' democracy had many iterations, with many people reforming its laws such as Solon and Cleisthenes.
+    	Originally, Athens was ruled by a line of kings, but, like many other Greek city states, it eventually became a republic. Athens' democracy had many iterations, with many people reforming its laws, such as Solon and Cleisthenes.
     	The primary actors in the democracy were the Athenian citizens, men over the age of twenty. Citizens were appointed, elected, or chosen by lot to positions of magistrate, tribunal, or council and could attend the assembly, which was the primary place of government.
-    	Democracy in ancient Athens was semi-stable, every so often a tyrant would overthrow it, usually to the oppression of the populace. To try to protect themselves against it, the Athenians began the practice of ostracism. Every year the Athenian people decided whether to conduct an ostracism. If so, the citizens of Athens would write on shards of broken pottery the name of a politician who they thought was getting too much power. Whoever had the most votes was exiled for ten years.
+    	Democracy in ancient Athens was semi-stable; every so often, a tyrant would overthrow it, and usually proceeded to make the lives of the people very difficult. To try to protect themselves against it, the Athenians began the practice of ostracism. Every year, the Athenian people decided whether to conduct an ostracism. If so, the citizens of Athens would write on shards of broken pottery the name of a politician who they thought was getting too much power. Whoever had the most votes was exiled for ten years.
```

Comment:
I like "to the oppression of the populace" better actually - it's shorter and more precise. But I'm ok with changing it if you want.

Reply:
I think the new way has a bit more character. We can afford a few extra words, I think.

Reply:
Ok, I'm fine with that too.